app/src/main/java/com/gusta/wakemehome/database/AlarmEntry.java

```diff
@@ -47,6 +47,19 @@ public AlarmEntry(int id, String location, double latitude, double longitude, do
         this.alert = alert;
     }
 
+    public AlarmEntry(AlarmEntry alarm) {
```

Comment:
Why would you use it instead of "public void setEnabled"? That's not a good practice to just copy another object in the contractor - if needed, a copyFrom method is better.